src/Nethermind/Nethermind.EofParser/Program.cs

```diff
@@ -0,0 +1,52 @@
+// SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+using Nethermind.Core.Extensions;
+using Nethermind.Evm.EOF;
+string line;
+
+int idx = 1;
+EvmObjectFormat.Logger = new InMemoryLogger
+{
+    IsTrace = true
+};
+
+while ((line = Console.ReadLine()) != null)
+{
+    HandleLine(idx++, line);
+}
+
+void HandleLine(int idx, string line)
+{
+    if (line.StartsWith("#"))
+    {
+        return;
+    }
+
+    line = new string(line.Where(c => char.IsLetterOrDigit(c)).ToArray());
```

Comment:
what other characters do we expect in input?

Reply:
There was a discussion where they decided to add special characters to separate sections..., it was easier to just remove all unaccepted chars in a bytecode

Reply:
Can you add a comment for that?

Reply:
What if it has comments, starting with `#` but after some bytecode?

Reply:
As per agreement comments aren't mixed with bytecodes in line so each line is either a comment | bytecode